playground/css/async/diamond.js

```diff
@@ -0,0 +1,16 @@
+// This creates a diamond dependency:
+// main (this file)
+//   -> chunk-a -> shared-base
+//   -> chunk-b -> shared-base
+//               -> chunk-a -> shared-base
+//
+// Expected CSS order: shared-base.css, chunk-a.css, chunk-b.css
```

Comment:
This is not the expected order. There's **no** expected order in this case. It depends on which file is sent to the browser first.